CMakeLists.txt

```diff
@@ -1,9 +1,12 @@
 cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
 
-# CMAKE
-# apt install build-essential git
-# git clone https://github.com/Kitware/CMake/; cd CMake
-# ./bootstrap && make && sudo make install
+# Set the C++ standard to C++17
+# set(CMAKE_CXX_STANDARD 17)
```

Comment:
This is already in the source folder cmakelists

Reply:
I just removed the comments